src/Controller/Admin/PageExportController.php

```diff
@@ -0,0 +1,62 @@
+<?php declare(strict_types=1);
+
+namespace Neusta\Pimcore\ImportExportBundle\Controller\Admin;
+
+use Neusta\Pimcore\ImportExportBundle\Documents\Export\PageExporter;
+use Neusta\Pimcore\ImportExportBundle\Toolbox\Repository\PageRepository;
+use Pimcore\Controller\UserAwareController;
+use Pimcore\Model\Document\Page;
+use Symfony\Component\HttpFoundation\HeaderUtils;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+final class PageExportController extends UserAwareController
+{
+    public function __construct(
+        private PageExporter $pageExporter,
+        private PageRepository $pageRepository,
+    ) {
+    }
+
+    /**
+     * @Route(
+     *     "/admin/neusta/import-export/page/export",
+     *     name="neusta_pimcore_import_export_page_export",
+     *     methods={"GET"}
+     * )
+     */
+    public function exportPage(Request $request): Response
+    {
+        $pageId = $request->query->getInt('page_id');
+        $page = $this->pageRepository->getById($pageId);
+
+        if (!$page instanceof Page) {
+            return new JsonResponse(
+                \sprintf('Page with id "%s" was not found', $pageId),
+                Response::HTTP_NOT_FOUND,
+            );
+        }
+
+        try {
+            $json = $this->pageExporter->toYaml($page);
```

Comment:
This return YAML, not JSON, right?

```suggestion
            $yaml = $this->pageExporter->toYaml($page);
```